src/common/helpers/pagination.helper.ts

```diff
@@ -20,6 +20,10 @@ export default class PaginationHelper<T> {
     this.totalPages = Math.ceil(this.elements.length / this.elementsOnPage);
   }
 
+  public get currentPage() {
+    return this.createPage()[0];
+  }
```

Comment:
We don't need that. Typically you want to itterate on array that returned by createPage so you can render whole page. In case of our profile I was getting [0] because we had only 1 element. 

So basically you want to do stuff like that outside of the handler